app/controllers/events/view/sessions.js

```diff
@@ -0,0 +1,12 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  currentRouteName: computed('session.currentRouteName', function() {
```

Comment:
this computed property returns a boolean. The name `currentRouteName` is counter-intuitive for that.